core/amber/src/main/scala/edu/uci/ics/texera/web/resource/ResultDownloadResource.scala

```diff
@@ -28,7 +24,7 @@ import scala.util.{Failure, Success, Try}
 
 object ResultDownloadResource {
 
-  private final val UPLOAD_BATCH_SIZE = 100
+  private final val UPLOAD_BATCH_SIZE = 10000
```

Comment:
better to add a comment, batch size in rows, or in cells, or in bytes?

Reply:
done